__tests__/app.test.js

```diff
@@ -178,17 +173,124 @@ describe("Post /api/articles/1/comments", () => {
         expect(response.body.comment).toEqual(outPut);
       });
   });
-  test.only("Should return with the a bad request if username missing", () => {
+  test("Should return with the a bad request if username missing", () => {
     return request(app)
       .post("/api/articles/1/comments")
       .send({
-        article_id: 1,
-
         body: "Gives a good insight in living under the pressure to achieve the same as a prodecessor or to be better",
       })
       .expect(400)
       .then(({ text }) => {
-        expect(text).toEqual('null value in column "author"');
+        expect(text).toEqual("null value");
+      });
+  });
+  test("Should return with the a 404 if id non existent", () => {
+    return request(app)
+      .post("/api/articles/100/comments")
+      .send({
+        userName: "rogersop",
+        body: "Gives a good insight in living under the pressure to achieve the same as a prodecessor or to be better",
+      })
+      .expect(400)
+      .then(({ text }) => {
+        expect(text).toEqual("Null value");
+      });
+  });
+  test("Should return with the a 400 if id is not valid", () => {
+    return request(app)
+      .post("/api/articles/hi/comments")
+      .send({
+        userName: "rogersop",
+        body: "Gives a good insight in living under the pressure to achieve the same as a prodecessor or to be better",
+      })
+      .expect(400)
+      .then(({ text }) => {
+        expect(text).toEqual("Not valid");
+      });
+  });
+  test("Should return with 201 if extra properties exist", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        userName: "rogersop",
+        body: "Gives a good insight in living under the pressure to achieve the same as a prodecessor or to be better",
+        votes: 10,
+      })
+      .expect(201)
+      .then((response) => {
+        const outPut =
+          "Gives a good insight in living under the pressure to achieve the same as a prodecessor or to be better";
+        expect(response.body.comment).toEqual(outPut);
```

Comment:
You want to make sure that all the properties you are expecting are returned here, please include the username too 👀